exercises/currying/Curry1.hs

```diff
@@ -0,0 +1,49 @@
+module Curry1 where
+
+-- I AM NOT DONE
+
+{-
+- In Haskell, functions that take more than one argument are usually
+  curried, meaning the function takes one argument and returns another function that takes the second argument. 
+  
+- The standard library defines a *higher order function* *curry* which 
+  takes a function from a pair (a,b) -> c as an argument and returns a function that takes a and returns a function b -> c.
+  
+curry :: ((a, b) -> c) -> a -> b -> c
+
+- The inverse of *curry* is called *uncurry*.
+
+uncurry :: (a -> b -> c) -> (a, b) -> c
+
+-}
+
+-- The manhatten distance of two coordinat points is defined as the distance in x + the distance in y. 
+-- This function takes the xdistance and ydistance and calculates the manhatten distance:
+
+
+
+manhattenDistance :: Int -> Int -> Int
```

Comment:
I believe Manhatt**a**n is spelled with an "a" towards the end, rather than an "e" (`/s/manhatten/manhattan/g`)

https://xlinux.nist.gov/dads/HTML/manhattanDistance.html#:~:text=Definition%3A%20The%20distance%20between%20two,y1%20%2D%20y2%7C.